website/docs/r/oraclepaas_mysql_service_instance.html.markdown

```diff
@@ -67,12 +68,14 @@ The following arguments are supported:
 
 * `availability_domain` - (Optional) Name of the availability domain within the region where the Oracle Database Cloud Service instance is to be provisioned. This is applicable only if you wish to provision to an OCI instance.
 
-* `enable_notification` - (Optional) Specifies if you would like to receive notification emails. The default is `true`
-
 * `notification_email` - (Optional) The email address to send notifications around successful or unsuccessful completions of the instance-creation operation.
 
 * `ip_network` - (Optional) This attribute is only applicable to accounts where regions are supported. The three-part name of an IP network to which the service instance is added. For example: /Compute-identity_domain/user/object
 
+* `shape` - (Optional) The desired compute shape.  A shape defines the number of Oracle Compute Units (OCPUs) and amount of memory (RAM). See [About Shapes](http://www.oracle.com/pls/topic/lookup?ctx=cloud&id=OCSUG210) in _Using Oracle Compute Cloud Service_ for more information about shapes.

```

Comment:
`Defaults to [value]`

Reply:
Taking @scross01 comment above to remove the default and make this required